src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -867,6 +867,31 @@ public enum VerticalLineAlignment
         Bottom
     }
 
+    /// <summary>
+    /// An enum of ellipsis location.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
```

Comment:
Please add this. it should be hidden before the API is opened.
[EditorBrowsable(EditorBrowsableState.Never)]

Note: https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.editorbrowsableattribute?view=net-5.0

Reply:
Done